release/util/download_wheels.sh

```diff
@@ -35,17 +35,19 @@ download_wheel "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERS
 
 # macOS.
 echo "Downloading Ray core MacOS wheels (x86_64)"
-download_wheel "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp37-cp37m-macosx_10_15_x86_64.whl"
+download_wheel "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp37-cp37-macosx_10_15_x86_64.whl"
```

Comment:
will test it manually once mac wheels are built on 2.5.1

Reply:
noob question: what does the change from `cp37m` to `cp37` mean?

Reply:
we were generating wheels that had "m" in the wheel name (using the system installed python) before this PR https://github.com/ray-project/ray/pull/36185
And we no longer generate wheels with "m" when we use conda. 

But I am also not sure why "m" in the first place.